### PR TITLE
chore(release): Release 2.2.0

### PR DESCRIPTION
**Pull request by mjc1283**

### CODE REVIEW

**Comment by mjc1283** [comment]
build-e2e

**Comment by mjc1283** [comment]
build-e2e

File: packages/optimizely-sdk/CHANGELOG.MD [hunk 1]
```diff
@@ -4,6 +4,17 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [2.2.0] - September 25, 2018
```
Comment (mikeng13) [inline packages/optimizely-sdk/CHANGELOG.MD]: Just need to update the date here

**Comment by mjc1283** [comment]
build-e2e

**Pull request merged**